### ISSUE TITLE
highlight around arrows+magnet should have more dilated space

### ISSUE THREAD

**Issue by zepumph** [phetsims/faradays-law#84]
We can use the new `FocusHighlightPath.getDilationCoefficient( node)` to do this easily (hopefully).
